src/kOS/Communication/StockConnectivityManager.cs

```diff
@@ -46,5 +54,18 @@ public bool HasConnection(Vessel vessel1, Vessel vessel2)
         {
             return true;
         }
+
+        public void AddAutopilotHook(Vessel vessel, FlightInputCallback hook)
+        {
+            // removing the callback if not already added doesn't throw an error
+            // but adding it a 2nd time will result in 2 calls.  Remove to be safe.
+            vessel.OnPreAutopilotUpdate -= hook;
+            vessel.OnPreAutopilotUpdate += hook;
```

Comment:
The RT manager does not subscribe to the vessel event because RT surpresses all control modifications made inside of the autopilot events in favor of it's own `AddSanctionedPilot` method.  See https://github.com/KSP-KOS/KOS/pull/1850/files#diff-e76af280dcf349ffd6916450b3aa238fR79